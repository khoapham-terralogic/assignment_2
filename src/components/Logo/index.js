import React from 'react';

import { logo } from '../../constants'
import { NavLink } from 'react-router-dom';

const Logo = () => {
    return (
        <div className="logo" data-testid="logo">
            <NavLink to="/"><img className="logo-img" src={logo} alt="logo" /></NavLink>
            <p className="logo-title" data-testid="logo-title">start your personal photo experience</p>
        </div>
    );
}

export default Logo;
