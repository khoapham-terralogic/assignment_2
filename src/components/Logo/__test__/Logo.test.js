
import React from 'react';
import Logo from '..'
import { shallow } from 'enzyme';

describe('<Logo />', () => {
    const wrapper = shallow(<Logo />)
    it('It should render', () => {
        expect(wrapper).toMatchSnapshot()
    })
    it('It should render properly', () => {
        expect(wrapper.find('[data-testid="logo-title"]').text()).toEqual("start your personal photo experience")
    })
});
