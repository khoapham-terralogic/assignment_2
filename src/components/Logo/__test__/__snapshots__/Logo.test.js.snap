// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> It should render 1`] = `
<div
  className="logo"
  data-testid="logo"
>
  <NavLink
    to="/"
  >
    <img
      alt="logo"
      className="logo-img"
      src="brand-logo.svg"
    />
  </NavLink>
  <p
    className="logo-title"
    data-testid="logo-title"
  >
    start your personal photo experience
  </p>
</div>
`;
